src/components/Suggestions.js

```diff
@@ -84,8 +85,16 @@ class Suggestions extends Component {
     return query.length >= minQueryLength && isFocused;
   };
 
+  renderSuggestion = (item, query) => {
+    if (this.props.renderSuggestion) {
```

Comment:
can you destruct `renderSuggestion` and check here `typeof renderSuggestion === 'function'`?

Reply:
done